modules/fedimint-mint-client/src/backup/recovery.rs

```diff
@@ -247,6 +258,16 @@ impl From<CompressedBlindedMessage> for BlindedMessage {
     }
 }
 
+#[derive(Debug, Clone, Decodable, Encodable)]
+pub enum MintRecoveryState {
+    V0(MintRecoveryStateV0),
+    #[encodable_default]
+    Default {
+        variant: u64,
+        bytes: Vec<u8>,
+    },
+}
```

Comment:
Pretty sure this requires a DB migration

Reply:
Adding a field already does unfortunately.